activesupport/CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
+Add `logger` as a dependency since it is a bundled gem candidate for Ruby 3.5
+
+*Earlopain*
```

Comment:
Could you fix?

https://buildkite.com/rails/rails/builds/108043#018fe820-3592-43e8-9bad-25785f217dd9/1272

```
activesupport/CHANGELOG.md:1 CHANGELOG header must start with '*' and 3 spaces
Add `logger` as a dependency since it is a bundled gem candidate for Ruby 3.5
^^^^
activesupport/CHANGELOG.md:3 CHANGELOG line must be indented 4 spaces
*Earlopain*
^^^^
13 changelogs inspected, 2 offenses detected
```

Reply:
Yes, my bad. Not good about running this beforehand and you get no notification for non-gh action failures

Reply:
No worries, though not sure we need notifications, anyone else reviewing this PR should have noticed it.

I enjoy pointing these things out in hopes it unblocks others, CI is often a source of confusion ranging from "why is this flaky test failing" to just completely ignoring it.

Reply:
Nah, I just mean the buildin notifications from github itself. If a github action fails you get info via email and push notificaion on mobile. It's per user _and_ repo, so everyone can set it up how they like it, unfortunatly this doesn't happen for external services which is sad (but somewhat understandable).

I have the bad habit of pushing and letting CI (not) notify me of things, especially since the rails suite is so big